ipv4 and ipv6 support + query timeout

gingerwizard opened the pull request.

bossinc commented on pkg/converters/converters.go:
```diff
@@ -352,3 +368,33 @@ func bigIntNullableConvert(in interface{}) (interface{}, error) {
+
+func ipConverter(in interface{}) (interface{}, error) {
+	if in == nil {
+		return (*string)(nil), nil
```
is this right? Why do we not return just`nil`?

gingerwizard commented:
@bossinc addressed comment.

Also based on user feedback, I separated query timeout from dial timeout. The default for the former needs to be a lot higher and we should decouple otherwise `Save and Test` behaviour will be poor.

![image](https://user-images.githubusercontent.com/12695796/187908706-c7094210-64ac-42c2-b5ca-1f180ea0121c.png)
